amazon_cloud_code_generator/cmd/refresh_modules.py

```diff
@@ -258,6 +268,7 @@ def renderer(self, target_dir: str, next_version: str):
         arguments = generate_argument_spec(documentation["options"])
         documentation_to_string = format_documentation(documentation)
         required_if = gen_required_if(self.schema)
+        mutually_exclusive = gen_required_if(self.schema)
```

Comment:
Is this supposed to be `get_mutually_exclusive()`? That function is never used, so I'm assuming it's supposed to be used here.

Reply:
Yes, thanks!